src/src/PluginStructs/P087_data_struct.h

```diff
@@ -9,12 +9,16 @@
 #include <Regexp.h>
 
 
-# define P087_REGEX_POS          0
-# define P087_NR_CHAR_USE_POS    1
-# define P087_FILTER_OFF_WINDOW_POS 2
-# define P087_MATCH_TYPE_POS 3
+# define P087_HEX_DATA_LEN_POS   0
+# define P087_HEX_HEADER_POS     1
+# define P087_HEX_DATA_LEN_ADD_POS  2
 
-# define P087_FIRST_FILTER_POS   6
+# define P087_REGEX_POS          3
+# define P087_NR_CHAR_USE_POS    4
+# define P087_FILTER_OFF_WINDOW_POS 5
+# define P087_MATCH_TYPE_POS 6
+
+# define P087_FIRST_FILTER_POS   9
```

Comment:
Not sure if this will break any existing config out there.
On the other hand, also not sure if any is (still) using this plugin anyway.